src/components/ChangelogDialog/__stories__/ChangelogDialog.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {Meta, Story} from '@storybook/react/types-6-0';
+
+import {Button} from '../../Button';
+import {ChangelogDialog, ChangelogDialogProps} from '../ChangelogDialog';
+import {ChangelogItem} from '../types';
+
+export default {
+    title: 'Components/ChangelogDialog',
+    component: ChangelogDialog,
```

Comment:
There is `actions` control to display callbacks in Action tab of story

```suggestion
    component: ChangelogDialog,
    argTypes: {
        onStoryClick: {
            action: 'onStoryClick'
        }
    }
```